commit 5460397b751150a6d1e1b42caebb428418ab5682
Author: Purna Chandra Mandal <purna.mandal@microchip.com>
Date:   Thu Jun 2 14:51:42 2016 +0530

    MIPS: pic32mzda: Fix linker error for pic32_get_pbclk()
    
    commit a726f1d2dd4fee179aa4513176d688ad309de6cc upstream.
    
    Early clock API pic32_get_pbclk() is defined in early_clk.c and used by
    time.c and early_console.c. When CONFIG_EARLY_PRINTK isn't set,
    early_clk.c isn't compiled and time.c fails to link.
    
    Fix it by compiling early_clk.c always. Also sort files in alphabetical
    order.
    
    Fixes: 6e4ad1b41360 ("MIPS: pic32mzda: fix getting timer clock rate.")
    Reported-by: Harvey Hunt <harvey.hunt@imgtec.com>
    Signed-off-by: Purna Chandra Mandal <purna.mandal@microchip.com>
    Reviewed-by: Harvey Hunt <harvey.hunt@imgtec.com>
    Cc: Ralf Baechle <ralf@linux-mips.org>
    Cc: Joshua Henderson <digitalpeer@digitalpeer.com>
    Cc: linux-mips@linux-mips.org
    Patchwork: https://patchwork.linux-mips.org/patch/13383/
    Signed-off-by: James Hogan <james.hogan@imgtec.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1209_5460397b751150a6d1e1b42caebb428418ab5682.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1209_5460397b751150a6d1e1b42caebb428418ab5682.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
